Tests/NumLib/TestODEInt.cpp

```diff
@@ -68,189 +87,139 @@ class TestOutput
         const double delta_t = (num_timesteps == 0) ? -1.0
                                                     : ((t_end-t0) / num_timesteps);
 
-        INFO("Running test %s with %u timesteps of size %g s.",
-             _file_name_part.c_str(), num_timesteps, delta_t);
-        init_file(delta_t);
+        INFO("Running test with %u timesteps of size %g s.", num_timesteps,
```

Comment:
I even would suggest to disable logog output for these tests (especially the linear solver output); The twelve NumLibODEInt* tests produced 58644 lines of output (instead of expected ~24).